check_commits.sh

```diff
@@ -90,15 +90,15 @@ do
         continue
     fi
 
-    if [ ! -z $CHECK_SIGNOFFS ]; then
+    if [ -n ${CHECK_SIGNOFFS} ]; then
         # Check that Signed-off-by tags are present.
         if ! echo "$COMMIT_MSG" | grep -F "Signed-off-by: ${COMMIT_USER_EMAIL}" >/dev/null; then
             echo >&2 "Commit ${i} is not signed off! Use --signoff with your commit."
             notvalid="$notvalid $i"
         fi
     fi
 
-    if [ ! -z $CHECK_CHANGELOGS ]; then
+    if [ -n ${CHECK_CHANGELOGS} -a ${COMMIT_USER_EMAIL} != "support@dependabot.com" ]; then
```

Comment:
This is not the most robust solution; a regular user could set such an email in his/her Git configuration and get commits without changelog. I just don't think that the potential harm of that is worth the time in making this too fancy...

Reply:
I think that is gripping for straws. It still has to go through us also.